src/frontend/src/components/authenticateBox.ts

```diff
@@ -562,12 +563,22 @@ export const authnPages = (i18n: I18n, props: AuthnTemplates) => {
   const templates = authnTemplates(i18n, props);
 
   return {
-    firstTime: (opts: Parameters<typeof templates.firstTime>[0]) =>
-      page(templates.firstTime(opts)),
+    firstTime: (
+      opts: Parameters<typeof templates.firstTime>[0],
+      // TODO: remove this parameter once the landing page is ready.
+      landingPageTemplate: boolean = false
```

Comment:
I needed this to use it in the showcase for now.

In a following PR, I'll remove it and the `firstTime` and `pick` will always come with the new landing page, whereas the `useExisting` comes with the old layout.